rpc/wrtc_call_queue_mongodb.go

```diff
@@ -1180,16 +1208,24 @@ func (queue *mongoDBWebRTCCallQueue) SendOfferInit(
 		defer cleanup()
 		defer close(answererResponses)
 
+		var finalResult string
+		defer func() {
+			duration := time.Since(call.StartedAt).Seconds()
+			callExchangeDuration.Observe(duration, sdkType, organizationID, finalResult)
```

Comment:
I _suspect_ you are still measuring only a subset of the call exchange with this value. Before I dive further into theories, what values are you seeing for `duration` in testing? What about same-subnet vs. different-subnet connections?

Reply:
is that different from when the caller gets a response and is "done"? basically I'm asking why the places we're adding the metric don't match 1:1 to where we're incrementing for the connection success/failure result